src/main/java/net/minecraftforge/client/event/ClientChatEvent.java

```diff
@@ -56,7 +54,7 @@ public void setMessage(String message)
     }
 
     /**
-     * {@return the original message that was to be sent to the server}
+     * {@return the original message that was to be sent to the server. This cannot be changed by mods}
```

Comment:
The first one has an ending period and this one doesn't. I also don't know how it works with the online return tag, perhaps check the javadoc tool?